.travis.yml

```diff
@@ -68,21 +66,21 @@ before_install:
     fi
   - |
     # Configure mocha for testing
-    if   node_version_lt '0.10'; then npm_use_module 'mocha' '2.5.3'
+    if   node_version_lt '0.12'; then npm_use_module 'mocha' '2.5.3'
     elif node_version_lt '4.0' ; then npm_use_module 'mocha' '3.5.3'
     elif node_version_lt '6.0' ; then npm_use_module 'mocha' '5.2.0'
     elif node_version_lt '8.0' ; then npm_use_module 'mocha' '6.2.2'
     fi
   - |
     # Configure nyc for coverage
-    if   node_version_lt '0.10'; then npm_remove_module_re '^nyc$'
+    if   node_version_lt '0.12'; then npm_remove_module_re '^nyc$'
     elif node_version_lt '4.0' ; then npm_use_module 'nyc' '10.3.2'
     elif node_version_lt '6.0' ; then npm_use_module 'nyc' '11.9.0'
     elif node_version_lt '8.0' ; then npm_use_module 'nyc' '14.1.1'
     fi
   - |
     # Configure supertest for http calls
-    if   node_version_lt '0.10'; then npm_use_module 'supertest' '1.1.0'
+    if   node_version_lt '0.12'; then npm_use_module 'supertest' '1.1.0'
```

Comment:
Since you removed all less than 0.12 from the CI, this line can just be removed.